MoEngage Actions Updated Doc (#3405)
segmentio/segment-docs: src/connections/destinations/catalog/actions-moengage/index.md

Include additional information that you think will be useful to the user here. For information that is specific to an individual mapping, please add that as a comment so that the Segment docs team can include it in the auto-generated content for that mapping.
-->

### Default Mapping

> > warning "Do not change the preset values"

Reviewer: ```suggestion
> warning ""
> Don't change the preset values
```

> > We already provide preset values to all mappings and hence no changes are required in the Identify User Actions, and Track Event Actions.

Reviewer: @aekanshd who provides the preset values? Segment or MoEngage?

Author: MoEngage provides the present values.

> > We already provide preset values to all mappings and hence no changes are required in the Identify User Actions, and Track Event Actions.

Reviewer: ```suggestion
> MoEngage provides preset values to all mappings. You don't need to make any changes to the Identify User Actions and Track Event Actions.
```
